README.md

```diff
@@ -1 +1,153 @@
-# booking_api
\ No newline at end of file
+# 📗 Table of Contents
```

Comment:
# Frontend review

- [ ] Please, kindly refactor this component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Header= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪


![image](https://github.com/gealsanchez/booking_api/assets/57421823/951a8206-ee9f-4b52-99a2-ea8ae1511baf)

- [ ] The app is trying to fetch tickets, however this feature has not been implemented:

![image](https://github.com/gealsanchez/booking_api/assets/57421823/23fb8c90-dd41-4ebe-bb97-f49cf1690d8f)

Kindly, remove this functionality since this has not been implemented (there is no route on the server). 👍

- [ ] The cars details sections do not look similar to the design given:

![image](https://github.com/gealsanchez/booking_api/assets/57421823/5824df67-c585-4d7b-a11e-e0ed6c95e2e1)

Kindly, touch up your styling a little so they look more like the design. 👍  

- [ ] Kindly, remove the old commented code found in:

1. SignUp.js
2. SignIn.js
3. Sidebar.js